microsoft-edge/webview2/concepts/print.md

```diff
@@ -0,0 +1,736 @@
+          {
+            std::wstring message = L"";
+            if (errorCode == S_OK && printStatus == COREWEBVIEW2_PRINT_STATUS_SUCCEEDED)
+            {
+              message = L"Printing " + std::wstring(title.get()) +
+                              L" document to printer is succedded";
+            }
+            else if (errorCode == S_OK && printStatus == COREWEBVIEW2_PRINT_STATUS_PRINTER_UNAVAILABLE)
+            {
+              message = L"Selected printer is not found, not available, offline or "
+                              L"error state.";
+            }
+            else if (errorCode == E_INVALIDARG)
+            {
+              message = L"Invalid settings provided for the specified printer";
+            }
+            else if (errorCode == E_ABORT)
+            {
+              message = L"Printing " + std::wstring(title.get()) +
+                                  L" document already in progress";
+            }
+            else
+            {
+              message = L"Printing " + std::wstring(title.get()) +
+                                  L" document to printer is failed";
+            }
+
+            AsyncMessageBox(message, L"Print to printer");
+
+            return S_OK;
+          })
+          .Get()));
+  return true;
+}
+```
+
+**APIs:**
+
+* [ICoreWebView2_16::Print() method](/microsoft-edge/webview2/reference/win32/icorewebview2_16#print)
+* [ICoreWebView2PrintSettings interace](microsoft-edge/webview2/reference/win32/icorewebview2printsettings1)
+* [ICoreWebView2PrintSettings2 interace](microsoft-edge/webview2/reference/win32/icorewebview2printsettings2)
+* [COREWEBVIEW2_PRINT_STATUS Enum](/microsoft-edge/webview2/reference/win32/icorewebview2_16#corewebview2_print_status)
+
+---
+
+<!-- end of tab-set -->
+
+
+<!-- ====================================================================== -->
+## PrintToPdf
+
+Silently print the current top-level document in the WebView2 to a PDF file. To completely control how printing is performed, you can print to a PDF and then build your own code to print the PDF. This API consists of an asynchronous PrintToPdf method and a PrintSettings object. The PrintToPdf method accepts a path that the PDF file should be saved to.
+
+<!-- ------------------------------ -->
+
+# [C#](#tab/csharp)
+
+[CoreWebView2.PrintToPdfAsync(resultFilePath, printSettings) Method](/dotnet/api/microsoft.web.webview2.core.corewebview2.printtopdfasync#microsoft-web-webview2-core-corewebview2-printtopdfasync)
+[CoreWebView2PrintSettings Class](/dotnet/api/microsoft.web.webview2.core.corewebview2printsettings)
+
+
+<!-- ------------------------------ -->
+
+# [C++](#tab/cpp)
+
+[ICoreWebView2_7::PrintToPdf(resultFilePath, printSettings, handler) method](/microsoft-edge/webview2/reference/win32/icorewebview2_7#printtopdf)
+[ICoreWebView2PrintSettings interface](microsoft-edge/webview2/reference/win32/icorewebview2printsettings)
+
+---
+
+<!-- end of tab-set -->
+
+### Example: PrintToPdf
```

Comment:
2 aspects to consider applying: 
* Possibly use only h2/h4/h6 in entire .md.  Repo is moving that direction eg Overview of APIs article.  Entered a global comment so maybe I shouldn't make that change here.
* Make "Example" headings match the parent scenario wording.
```suggestion
#### Example: The PrintToPdf method
```